src/common/components/Button/Button.module.scss

```diff
@@ -0,0 +1,36 @@
+.button {
+  padding: 5px 10px;
+  font-family: "Roboto";
+  font-size: 1rem;
+  border: 1px solid #aaaba8;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: all 0.1s linear;
+
+  &:disabled {
+    opacity: 0.3;
+    cursor: not-allowed;
+  }
+}
+
+.green {
+  color: #fff;
+  background-color: #058240;

```

Comment:
Would be nice to create variables file and save colors there

Reply:
Done - create color variables in variables.scss file